README.md

```diff
@@ -1251,6 +1251,16 @@ Default: `1`
 
     let g:ycm_auto_stop_csharp_server = 1
 
+### The `g:ycm_csharp_server_port` option
+
+When g:ycm_auto_start_csharp_server set to `1`, specifies the port for 
```

Comment:
Typo, "is set".

I'd also change "uses the OS default" to "uses an unused port provided by the OS."